aspnetcore/migration/1x-to-2x/identity-2x.md

```diff
@@ -0,0 +1,272 @@
+---
+title: Migrating Auth and Identity to ASP.NET Core 2.0
+author: scottaddie
+description: This article outlines the most common steps for migrating ASP.NET Core 1.x authentication and Identity to ASP.NET Core 2.0.
+keywords: ASP.NET Core,Identity,authentication
+ms.author: scaddie
+manager: wpickett
+ms.date: 08/01/2017
+ms.topic: article
+ms.technology: aspnet
+ms.prod: asp.net-core
+uid: migration/identity-2x
+---
+
+# Migrating Authentication and Identity to ASP.NET Core 2.0
+
+By [Scott Addie](https://github.com/scottaddie) and [Hao Kung](https://github.com/HaoK)
+
+ASP.NET Core 2.0 has a new model for authentication and [Identity](xref:security/authentication/identity) which simplifies configuration by using services. ASP.NET Core 1.x applications that use authentication or Identity can be updated to use the new model as outlined below.
+
+<a name="auth-middleware"></a>
+
+## Authentication Middleware and Services
+In 1.x projects, authentication is configured via middleware. A middleware method is invoked for each authentication scheme you want to support.
+
+The following 1.x example configures Facebook authentication with Identity in *Startup.cs*:
+
+```csharp
+public void Configure(IApplicationBuilder app, ILoggerFactory loggerfactory) {
+    app.UseIdentity();
```

Comment:
@Tratcher Can we get this 1.x example updated in the announcement too? https://github.com/aspnet/Announcements/issues/262